UDV-Core/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "UDV-Core",
+  "version": "0.0.1",
```

Comment:
Please align version number with the ones of index.html and Demo.html (v0.3).

Reply:
Added a `doc/Tagging.md` documenting the tagging logic and process.